spec/models/employer_events/employer_importer_spec.rb

```diff
@@ -801,6 +903,20 @@
       expect(employer_record).to have_received(:employer_office_locations=).with(instance_of(Array))
     end
 
-  end
-end
+    it 'checks the event type' do
+      expect(subject.is_contact_information_update_event?).to eq false
+      expect(contact_changed_subject.is_contact_information_update_event?).to eq true
+      expect(address_changed_subject.is_contact_information_update_event?).to eq true
+    end
+
+    it 'extracts office location attributes' do 
```

Comment:
Try using a describe block here instead.
```suggestion
    describe '#extract_office_location_attributes' do 
```